README.md

```diff
@@ -112,7 +112,7 @@ describe('UserService', function () {
 });
 ```
 
-We're now setup to receive the mocked backend response. In this case, we'll get back an object with `name` equal to `Bill Gates` and his email too.
+We're now setup to receive the mocked backend response. Our response will have several properties, with the `data` property referring to the body of our response. In this case, It'l be an object with `name` equal to `Bill Gates` and his email too.
 
```

Comment:
Hi @mendelB looks great, but there's a typo - It'l should be It'll - mind fixing that and then mentioning me and I'll merge this in. Thanks!